<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cell_borrowClass</class>
 <widget class="QWidget" name="cell_borrowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cell_borrow</string>
  </property>
  <widget class="QTableWidget" name="borrowTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>961</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>650</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:grey;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>40</y>
     <width>215</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="searchLE"/>
    </item>
    <item>
     <widget class="QPushButton" name="searchButton">
      <property name="text">
       <string>查找</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/Book/search.svg</normalon>
       </iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>256</width>
     <height>76</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="BookMS.qrc">:/background/cnu.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="borrowLabel">
      <property name="styleSheet">
       <string notr="true">QLabel {

                          color: #333;
                          font-size: 30px;
                          font-weight: bold;
                          padding: 10px;
                          border-radius: 5px;
                          }</string>
      </property>
      <property name="text">
       <string>可借阅书籍</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BookMS.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>borrowTable</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>cell_borrowClass</receiver>
   <slot>onRightClickShowMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchButton</sender>
   <signal>clicked()</signal>
   <receiver>cell_borrowClass</receiver>
   <slot>onSearchAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onRightClickShowMenu(QPoint)</slot>
  <slot>onSearchAction()</slot>
 </slots>
</ui>
